packages/calcite-components/src/components/tile-group/tile-group.stories.ts

```diff
@@ -46,2304 +46,747 @@ export default {
+      <div class="child">${getTileGroupHtml({ contentTop: true }, layout, "none", "l")}</div>
+    </div>
+
+    <!-- content-bottom slotted images -->
+    <div class="parent">
+      <div class="child right-aligned-text">content-bottom slotted images</div>
+      <div class="child">${getTileGroupHtml({ contentBottom: true }, layout, "none", "s")}</div>
+      <div class="child">${getTileGroupHtml({ contentBottom: true }, layout, "none", "m")}</div>
+      <div class="child">${getTileGroupHtml({ contentBottom: true }, layout, "none", "l")}</div>
+    </div>
+
+    <!-- slotted images in both slots -->
+    <div class="parent">
+      <div class="child right-aligned-text">slotted images in both slots</div>
+      <div class="child">${getTileGroupHtml({ contentBottom: true, contentTop: true }, layout, "none", "s")}</div>
+      <div class="child">${getTileGroupHtml({ contentBottom: true, contentTop: true }, layout, "none", "m")}</div>
+      <div class="child">${getTileGroupHtml({ contentBottom: true, contentTop: true }, layout, "none", "l")}</div>
+    </div>
+  `;
+}
 
-  <!-- icon and heading (large visual) links -->
-  <div class="parent">
-    <div class="child right-aligned-text">icon and heading (large visual) links</div>
-    <div class="child">
-      <calcite-tile-group layout="vertical" scale="s">
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-      </calcite-tile-group>
-    </div>
-    <div class="child">
-      <calcite-tile-group layout="vertical">
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-      </calcite-tile-group>
-    </div>
-    <div class="child">
-      <calcite-tile-group layout="vertical" scale="l">
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-        <calcite-tile heading="Tile heading lorem ipsum" href="/" icon="layers"></calcite-tile>
-      </calcite-tile-group>
-    </div>
-  </div>
-`;
+export const allVariantsHorizontal = createVariantsHtmlStory("horizontal");
+export const allVariantsVertical = createVariantsHtmlStory("vertical");
```

Comment:
I think you broke Chromatic 😄 https://www.chromatic.com/test?appId=6266d45509d7eb004aa254fb&id=677ce7bbb76f8e6437051490

Reply:
Oh noes! @eriklharper Would a shorter description value help here? We could split the all-variants stories, but it sort of defeats the purpose. 🥲

Reply:
You'd have to just try shortening it I suppose and see it makes it small enough for Chromatic to capture, but yeah, possibly that could make a difference.